src/QueryM.purs

```diff
@@ -826,6 +839,19 @@ mkOgmiosRequest = mkRequest
   (listeners <<< _.ogmiosWs <$> ask)
   (underlyingWebSocket <<< _.ogmiosWs <$> ask)
 
+-- | Builds an Ogmios request action using `Aff`
+mkOgmiosRequestAff
```

Comment:
@ngua  is the intention to use this to get era summaries in Aff so we can add to our config?